### DISCUSSION TITLE
Gitleaks Discord Server

### DISCUSSION THREAD

**Discussion by zricethezav** [zricethezav/gitleaks#1169] [General]
Hi Everyone! We just started a Secret Scanning Discord Server.

You can join here for live discussions: https://discord.gg/sydS6AHTUP

**Comment by aeells** [comment]
Invalid link...

**Comment by anoopvlcy** [comment]
> Hi Everyone! We just started a Secret Scanning Discord Server.
> 
> You can join here for live discussions: https://discord.gg/sydS6AHTUP

@zricethezav  The link provided seems to be invalid. could you please share correct link?
